dev-libs/libfilezilla/libfilezilla-0.6.1.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="C++ library offering some basic functionality for platform-independent programs"
+DESCRIPTION="Some of the highlights include:
+
+- A typesafe, multi-threaded event system that's very simple to use yet \
+extremely efficient
+- Timers for periodic events
+- A datetime class that not only tracks timestamp but also their accuracy, \
+which simplifies dealing with timestamps originating from different sources
+- Simple process handling for spawning child processes with redirected I/O
+"
+HOMEPAGE="https://lib.filezilla-project.org"
+COPYRIGHT="2015-2016 Tim Kosse"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://sourceforge.net/projects/filezilla/files/libfilezilla/$portVersion/libfilezilla-$portVersion.tar.bz2"
```

Comment:
Well, it is not safer because SF redirects it back to http. But that's OK to use https, even if we get redirected to http.
That said, if you want a really short URL you can use:
```
SOURCE_URI="https://downloads.sf.net/filezilla/libfilezilla-0.6.1.tar.bz2"
```
(you can also make it one char smaller by dropping the s in https ;-)